src/orchard/inspect.clj

```diff
@@ -356,18 +372,68 @@
         (render-ln))
     inspector))
 
+(defn- rendered-element-str
```

Comment:
I really don't like having to inspect what we have rendered after the fact and parsing it. It is preferable to orchestrate everything beforehand so that we don't need to do this.

Reply:
I'm actually rendering the key before I render the separator and the map value (which currently need to know if the key spans multiple lines or not). What would be your suggestion instead? I kind of have to render the key to find out if it is a long key spanning multiple lines, no?

In another attempt, I looked back what we rendered from within `render-map-separator` and `render-map-value`, but I thought that is a bit more fragile.

Reply:
Yeah, rendering the key first is fine, but I suggest dragging around the information about being too long explicitly, not peek into the last value in `:rendered` to figure it out.

Reply:
I changed this and pass the values I need around.